lua/telescope/actions/init.lua

```diff
@@ -272,6 +272,23 @@ function actions.close_pum(_)
   end
 end
 
+--- Copy current entry command to a specific register or unnamed plus as default
+--- Don't let user use  blackhole register (_), let them choose instead
```

Comment:
Sorry, should have caught this earlier: could you please clarify this to 
Passing `_` as a register will always trigger asking for user input.